### PR TITLE
[WIP] listCollaborators

### PR DESCRIPTION
**Pull request by shunjikonishi**

### CODE REVIEW

**Comment by sukeshni** [comment]
@shunjikonishi Please review

**Comment by shunjikonishi** [comment]
@sukeshni fix me

I think the response structure of collaborator is different from user object.

See.

https://developer.github.com/v3/repos/collaborators/#response
- The field 'email', 'blog', and so on are not exist
- Maybe 'site_admin' is most important field of Collaborator. I guess it means repository owner or not.

I guess if you call User#email method in the test, it will fail.

You have to make Collaborator class in models.

**Comment by sukeshni** [comment]
@shunjikonishi I did tried it with Collaborator class in models, and thought the response should contain User and id, which were already present in User,so used it. 
I agree with you, it should be separate.

**Comment by sukeshni** [comment]
@shunjikonishi Please review

**Comment by shunjikonishi** [comment]
## Class name should be singular.

Collaborators -> Collaborator

## 'description' field

It doesn't exist in the response.
Why you added 'description' field to Collaborator? 

## 'site_admin' field

It shoud be Boolean

## CollaboratorsOpSpec

It has two tests.
But It seems to same.
What's the difference?
What did you mean?

## Model class should be test.

You have to call all methods of Collaborator class in test.
If you did so, you could find 'description' method cause a error.

@sukeshni fix me

File: src/test/scala/CollaboratorOpSpec.scala [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import org.scalatest.path.FunSpec
+import codecheck.github.models.Collaborator
+import scala.concurrent.Await
+
+class CollaboratorOpSpec extends FunSpec with Constants {
+
+  describe("listCollaborators") {
+    it("should return atleast one collaborator") {
+      val res = Await.result(api.listCollaborators(organization, repo), TIMEOUT)
+      assert(res.length >= 1)
```
Comment (shunjikonishi) [inline src/test/scala/CollaboratorOpSpec.scala:10]: Like this.

``` scala
val c = res(0)
assert(res.id > 0)
assert(res.login.length > 0)
assert(res.avatar_url.length > 0)
assert(res.url.length > 0)
assert(res.site_admin == false)
//This line cause error
assert(res.description.length >= 0)
```

Calling each methods detects following bug possibility.
- typo. - e.g. def id = get("idx") 
- field existence. (GitHub API document might be wrong.)

**Comment by sukeshni** [comment]
@shunjikonishi Please review me.

**Comment by shunjikonishi** [comment]
@sukeshni 

About indentation. 
Replace tab to 2 spaces. Other wise LGTM.
You can merge this after fixing indentation.

**Pull request merged**